File: Proyecto_Peluqueria/Proyecto_Peluqueria/src/main/java/Modelss/Reserva.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package Modelss;

import java.util.Date;
import lombok.Getter;
import lombok.Setter;

/**
 *
 * @author Luis
 */
@Getter
@Setter

public class Reserva {
    private int idReserva;
    private Date fecha;
    private Date hora;
    private String estado;
    private Cortes corte;
    private Cliente cliente;
    private Usuarios usuarios;
    
    
    

}

File: Peluqueria/src/main/webapp/js/reserva.js
/* 
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Other/javascript.js to edit this template
 */

$(document).ready(function(){
    $.ajax({
        url:"./Reserva",
        dataType:'json',
        type:'get',
        cache:false,
        success: function(data){
            console.log(data);
            var event_data = '';
            
    for(let pan of data){
        event_data += '<tr>'; 
                event_data += '<td>' + pan.cliente.nombre + '</td>'; 
                event_data += '<td>' + pan.cliente.correo + '</td>'; 
                event_data += '<td>' + pan.estado + '</td>'; 
               event_data += '<tr>'; 
    }

            console.log(data);
            $("#list_table_json").append(event_data);
        },error:function(d){
            console.log("error");
            alert("404 espere hasta que se cargue el archivo");
        }
    });
});
